bigtable/client/row_reader.cc

```diff
@@ -106,7 +147,7 @@ void RowReader::MakeRequest() {
   processed_chunks_count_ = 0;
 
   google::bigtable::v2::ReadRowsRequest request;
-  request.set_table_name(std::string(table_name_));
+  request.set_table_name(std::string(table_name_.get()));
```

Comment:
Why does the result from `.get()` need to be wrapped in a `std::string`?

Reply:
Fixed.